src/Server/SSH/SSHChannel.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <memory>
+
+struct ssh_channel_struct;
+struct ssh_session_struct;
+using ssh_channel = ssh_channel_struct *;
```

Comment:
same comments as for SSHBind

Reply:
Fixed in `Remove unused get methods and fix types naming for ssh objects`